### DISCUSSION TITLE
BOTS

### DISCUSSION THREAD

**Discussion by ShhubhamA** [deso-protocol/dips#76] [Q&A]
Any ideas what could be done to prevent the spamming from bots!?

Right now its 270 bots liking lol imagine if someone decides to release 2700 😮

It would drain the user experience !!

**Comment by matt-mcmarsh** [comment]
Have suggested
a "fix" on notifications
with a filter - i.e. I can specifiy only show notifications of accounts with x$ coin price
also only show notifications for accounts with x followers

**Reply by ItsAditya-xyz** [reply]
Doesn't sounds like an efficient way. They can easily increase their coin price and follower count to get through this barrier. Also, this would impact the visibility of new user (whose coin price and follower count is less)

**Comment by ItsAditya-xyz** [comment]
I don't know if there is any way to stop these bots unless you blacklist them globally. People talk about having captcha while account creation but all these are frontend limits. Other arguments include limiting of APIs but that would badly effect external apps built at the top of BitClout.

And above all you never know if they are running their own node to control the bots and if that's the case then blacklisting seems an only way to get rid of them for short time. But this also isn't an efficient way

**Comment by brootle** [comment]
@bluepartyhat don't add notifications from accounts that have coin price 0.00, easy fix

**Comment by tijno** [comment]
Is it worth consolidating this discussion in the earlier topic #65

**Reply by matt-mcmarsh** [reply]
just posted in 65
https://github.com/bitclout/cips/discussions/65#discussioncomment-1012251

**Reply by matt-mcmarsh** [reply]
It is more an improvement to notifications
but would also allow to avoid the annoyance of the bot notifications
